changelog/8789.feature.rst

```diff
@@ -0,0 +1 @@
+Support TOML v1.0.0 syntax in ``pyproject.toml``.
```

Comment:
This should point out that the `toml` dependency got replaced by `tomli` - something e.g. distribution packagers could easily miss and definitely should know about!

Reply:
:+1: Please have a look if the message now looks better to you